tests/constants.py

```diff
@@ -769,6 +777,43 @@
 INVALID_SANDBOX_API_INCORRECT_DOMAIN["base_url"] = "https:///api/"
 
 
+# Valid Application List API response
+
+VALID_APPLIST_API: Dict[str, Union[str, Dict[str, str], Path, bool]] = {}
+VALID_APPLIST_API["base_url"] = "https://analysiscenter.veracode.com/api/"
+VALID_APPLIST_API["version"] = {"getapplist.do": "5.0"}
+
+VALID_APPLIST_API["app_id"] = "1337"
+VALID_APPLIST_API["app_name"] = "TestApp"
+VALID_APPLIST_API["build_id"] = "v1.2.3"
+VALID_APPLIST_API["username"] = "TestUser"
+
+# https://help.veracode.com/reader/LMv_dtSHyb7iIxAQznC~9w/kb2SM9net26_L91VploQGw
+VALID_APPLIST_API_RESPONSE_XML = {}
```

Comment:
Replicate throughout.
```suggestion
VALID_UPLOAD_API_APPLIST_RESPONSE_XML = {}
```